library/src/interactor_impl.cxx

```diff
@@ -193,7 +198,16 @@ class interactor_impl::internals
         self->Window.PrintSceneDescription(log::VerboseLevel::INFO);
         break;
       default:
-        if (keySym == "Left")
+
+        if (keyCode == '2' || keySym == "KP_Down")
```

Comment:
what is kp down ?

Reply:
that's the symbol my linux distribution gives me when pressing the key "2" on the numpad...

Reply:
hum, not sure we want to support this.

Reply:
Not sure how to detect when key 2 is pressed then...

Reply:
Why not? If shortcut is "2" you can expect to use the 2 key of the numpad. But anyway, if we consider the non numpad key "2", VTK does not provide a keycode for it on my keyboard, and the sym is "ecute" which is the french e with accent on which the 2 stands but then it is not cross country... weired!

Reply:
Yeah, we also need to avoid non-international keys. tbh this design is not great, we may need to add a layer of abstraction.